index.js

```diff
@@ -14,9 +15,21 @@ CmdHandler.setVerbose();
 const handler = new CmdHandler('commands');
 handler.loadAllCommands();
 
-client.on('ready', () =>
+client.on('ready', () => {
+    /**
+     * When the bot is ready modify the parameters (username, activity ...)
+     */
+    let bot;
```

Comment:
```js
const bot = new Bot(client)
```
Pas besoin de `let` ici

Reply:
Je sais pas c'etait pdnt le dev j'ai fai sa

Reply:
Elles sont faites pour ça les reviews :sweat_smile:

Reply:
Ouais 🤣

Reply:
Bas fait l'edit stp:

https://github.com/hichxm/codefr_bot/edit/set-default-settings-of-the-bot/index.js?pr=/Guiz711/codefr_bot/pull/14

Reply:
Je suis occupé, je te laisse le faire :sweat_smile:

Reply:
:octocat: